primitives/state-machine/src/overlayed_changes/mod.rs

```diff
@@ -137,6 +160,10 @@ pub struct StorageChanges<Transaction, H: Hasher, N: BlockNumber> {
 	/// Phantom data for block number until change trie support no_std.
 	#[cfg(not(feature = "std"))]
 	pub _ph: sp_std::marker::PhantomData<N>,
+
+	/// Changes to the transaction index,
+	#[cfg(feature = "std")]
```

Comment:
It is unlikely that this will be useful in sandboxed environment

Reply:
yes always the 'no_std' is 'wasm' or sandbox heuristic, I seem to agree on https://github.com/paritytech/substrate/pull/7275 but it is no priority. 
I remember exposing some unused stuff in state-machine (change trie, but maybe in an unmerged pr) because it makes thing simplier, but I agree this is a non issue.